src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -235,5 +235,9 @@ class AnnotationSpec private constructor(builder: AnnotationSpec.Builder) {
     @JvmStatic fun builder(type: KClass<*>): Builder {
       return builder(type.asClassName())
     }
+
+    @JvmStatic fun throwsBuilder(): Builder {
```

Comment:
Not sure we need this. It's about the same as `builder<Throws>()` which we could add support for. Even `builder(Throws::class)` is short.